packages/components/scripts/templates/__tests__/component.ct.tsx

```diff
@@ -0,0 +1,24 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { toKebabCase } from '../../transforms';
+import { type ComponentFileBuilderResponse } from '../../types';
+
+export const Setup = (componentName: string): ComponentFileBuilderResponse => ({
+    content: `/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { expect, test } from '@playwright/experimental-ct-react';
+import { ${componentName} } from "../${componentName}";
```

Comment:
```suggestion
import { expect, test } from '@playwright/experimental-ct-react';

import { ${componentName} } from "../${componentName}";
```
Did you sync the template with the lint fix?

Reply:
not yet